packages/zephyr/src/theme/colors.ts

```diff
@@ -196,9 +195,8 @@ const robins = {
   robin4: '#FF6D00',
 };
 
-const singles = {
-  transparent: 'transparent',
-  transparentSafariHack: 'rgba(255, 255, 255, 0.001)',
+const miscellaneous = {
+  transparent: 'rgba(255, 255, 255, 0.001)',
```

Comment:
Can we add a comment like so:
```suggestion
  /** @see https://stackoverflow.com/questions/46309641/what-happens-in-safari-with-the-transparent-color */
  transparent: 'rgba(255, 255, 255, 0.001)',
```